plugins/forklift/README.md

```diff
@@ -4,10 +4,12 @@ Plugin for ForkLift, an FTP application for OS X.
 
 ### Requirements
 
-* [ForkLift](http://forkliftapp.com/forklift/)
+* [ForkLift](http://www.binarynights.com/forklift/)
 
 ### Usage
 
-* If `fl` is called without arguments then the current folder is opened in ForkLift. Is equivalent to `fl .`
+<code>fl *file*</code>
```

Comment:
This `file` is confusing, it should be folder or directory. The `<code>` should work with regular backticks. And perhaps putting the argument in brackets `[]` would underscore that it is an optional argument.